### DISCUSSION TITLE
[Feature] Auto merge Duplicate names under Facial Recognition

### DISCUSSION THREAD

**Discussion by Dvalin21** [immich-app/immich#3272] [Feature Request]
### The feature

First, AMAZING WORK!! Awesome app!!!  I noticed that after the system goes through FR and your able to add names, it doesn't merge multiple entries with the same name. Wondering if you this could be an added feature? Thanks in advance.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can merge people using the web interface, as noted in the release of version 1.67 https://github.com/immich-app/immich/releases/tag/v1.67.0

Face merging UI isn't available on the mobile app just yet
